sql/item_mods.sql

```diff
@@ -1672,7 +1672,7 @@ INSERT INTO `item_mods` VALUES (10543,1,27);  -- DEF: 27
 INSERT INTO `item_mods` VALUES (10543,14,12); -- CHR: 12
 INSERT INTO `item_mods` VALUES (10543,27,-6); -- ENMITY: -6
 INSERT INTO `item_mods` VALUES (10543,30,10); -- MACC: 10
-INSERT INTO `item_mods` VALUES (10543,455,7); -- SONG_SPELLCASTING_TIME: 7
+INSERT INTO `item_mods` VALUES (10543,833,7); -- Song recast delay -7
```

Comment:
This should be the enum name of the relevant mod

Reply:
I'm going to run tooling on this to see what comes up.  Creating a branch from this